18F/identity-idp
# Upgrade Webpack dependencies to resolve security advisories

## 🛠 Summary of changes

Upgrades Webpack-related dependencies to their latest versions.

Initially this was targeted at `webpack-dev-server`, which has some related open security advisories (GHSA-wr3j-pwj9-hqq6, GHSA-rv95-896h-c2vc, GHSA-cxjh-pqwp-8mfp), but I decided to upgrade a few others since `webpack-dev-server` at least required a version bump of `webpack-cli` to >5.0.

Related CHANGELOG:

- https://github.com/webpack/webpack-dev-server/blob/master/migration-v5.md
- https://github.com/webpack/webpack-cli/releases/tag/webpack-cli%405.0.0
- https://github.com/webpack/webpack/releases
- https://github.com/webdeveric/webpack-assets-manifest/releases

These security updates don't block builds since [we only check for dependencies installed in production environments](https://github.com/18F/identity-idp/blob/7ce91b5b777ca45f7ca4cddc212c7704b5638b46/Makefile#L89-L90), but it's prudent to resolve them, and minimizes noise in the GitHub UI and push messages..

## 📜 Testing Plan

1. `yarn install`
2. `make run`
3. Visit http://localhost:3000
4. Observe no errors in the browser or in the Terminal output
